samples/wifi/sta/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(nrf_wifi_sta)
+
+target_include_directories(app PUBLIC ${ZEPHYR_BASE}/subsys/net/ip)
```

Comment:
Can this not link with the zephyr networking stack in a cleaner way?

Reply:
@nordicjm  Observed that, the other samples are also doing the same as the header file is a private one.